BoxContentSDK/BoxContentSDK/Requests/BOXFolderItemsRequest.m

```diff
@@ -43,16 +45,24 @@ - (void)cancel
     }
 }
 
+- (void)setRangeStep:(NSUInteger)rangeStep
+{
+    _rangeStep = rangeStep;
+}
+
 - (NSUInteger)rangeStep
 {
-    NSUInteger rangeStep = 1000;
-    
-    // Root folder has better performance with a smaller page size
-    if ([self.folderID isEqualToString:BOXAPIFolderIDRoot]) {
-        rangeStep = 100;
+    if (_rangeStep == 0) {
+        NSUInteger rangeStep = 1000;
+
+        // Root folder has better performance with a smaller page size
```

Comment:
This isn't unique to root, is it? I think it's just that large/complex folders have better performance with smaller page sizes?